tools/SetupFlow/DevHome.SetupFlow/Views/AddRepoDialog.xaml

```diff
@@ -60,7 +66,15 @@
             <TextBlock x:Name="UrlErrorTextBlock" Style="{ThemeResource BaseTextBlockStyle}" Text="{x:Bind AddRepoViewModel.UrlParsingError, Mode=OneWay}" Visibility="{x:Bind AddRepoViewModel.ShouldShowUrlError, Mode=OneWay}"/>
 
             <!-- Log into account -->
-            <ComboBox x:Name="RepositoryProviderComboBox" Visibility="{x:Bind AddRepoViewModel.ShowAccountPage, Mode=OneWay}" HorizontalAlignment="Stretch" x:Uid="ms-resource:///DevHome.SetupFlow/Resources/RepositoryDisplay" ItemsSource="{x:Bind AddRepoViewModel.ProviderNames}" Margin="0, 20, 0, 0" >
+            <!-- Please keep SelectedIndex bound to AccountIndex.  This is used to reset the combobox to empty
+            when the user navigates to the account page.-->
+            <ComboBox x:Name="RepositoryProviderComboBox" 
+                      Visibility="{x:Bind AddRepoViewModel.ShowAccountPage, Mode=OneWay}"
+                      HorizontalAlignment="Stretch"
+                      x:Uid="ms-resource:///DevHome.SetupFlow/Resources/RepositoryDisplay"

```

Comment:
You should just need 
```suggestion
                      x:Uid="RepositoryDisplay"
```
Goes for all in this file.

Reply:
Sweet.  Thank you for the suggestion.